MAUI/TreeView/expand-collapse.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Expand and Collapse in .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about Expand and Collapse support in the Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
+platform: .NET MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Expand and Collapse in Xamarin TreeView (SfTreeView)
+
+The TreeView allows you to expand and collapse the nodes either by user interaction on the nodes or by programmatically. 
+
+##  Expand Action Target
+
+ Expanding and Collapsing of nodes can be performed either by tapping the expander view or in both expander view and content view by setting the `ExpandActionTarget` property.

```

Comment:
Expanding and collapsing of nodes can be performed by tapping the expander view or by setting the `ExpandActionTarget` property in both the expander view and content view.